proxies/pensjon-testdata-facade-proxy/config.yml

```diff
@@ -51,7 +51,7 @@ spec:
           cluster: dev-gcp
     outbound:
       external:
-        - host: popp-testdata.dev.intern.nav.no
+        - host: popp-testdata.intern.dev.nav.no
```

Comment:
Ja bra, det var disse to som hadde fått service discovery ved en feil forrige gang.